src/main/java/com/mx/iwaconsolti/calculator/CalculatorApplication.java

```diff
@@ -23,9 +28,30 @@ public void run(String... args) throws Exception {
 		System.out.print("Enter the second number: ");
 		int secondNumber = scanner.nextInt();
 
-		// Use the SumService to calculate the sum
-		int result = firstNumber + secondNumber;
-
-		System.out.println("The sum of " + firstNumber + " and " + secondNumber + " is: " + result);
+		String option;
+
+		do{
+			System.out.println("Select the operation to perform Sum, Minus, Multiply, Divide: ");
+			option = scanner.next().toLowerCase();
+
+			switch (option){
+				case "sum" :
+					System.out.println("the result of the sum is: " + calculatorService.sum(firstNumber, secondNumber));
+					break;
+				case "minus" :
+					System.out.println("the result of the minus is: " + calculatorService.minus(firstNumber, secondNumber));
+					break;
+				case "multiply" :
+					System.out.println("the result of the multiply is: " + calculatorService.multiply(firstNumber, secondNumber));
+					break;
+				case "divide" :
+					System.out.println("the result of the divide is: " + calculatorService.divide(firstNumber, secondNumber));
+					break;
+				default:
+					System.out.println("invalid option");
+			}
+		} while(!option.equals("sum") && !option.equals("minus") && !option.equals("multiply") && !option.equals("divide"));
```

Comment:
You can create a boolean that is changed to true if they enter in one of the cases, and is false if the default happen.

Reply:
ya es el cambio que se solicito